bHY6yh-d41894/cod-yyyiou_bHY6yh-d41894.xml

```diff
@@ -0,0 +1,234 @@
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="C" n="cod-yyyiou">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-12-24" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#C" n="57-r"/>
+            <cb ed="#C" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d41894"><!-- p2l6-r5 -->
+            <head xml:id="bHY6yh-d41894-Hd1e108">Rubrica 5</head>
+            <head xml:id="bHY6yh-d41894-Hd1e111" type="question-title">De tyrannide et tyranno</head>
+            <p xml:id="bHY6yh-d41894-d1e114">
+              De tyrampno et
+              <lb ed="#C"/>tyrampnide capitulum
+            </p>
+            <!--next paragraph begins on the same line as the previous paragraph-->
+            <p xml:id="bHY6yh-d41894-d1e119">
+              <ref xml:id="bHY6yh-d41894-Rd1e115">IOb his<lb ed="#C" break="no"/>toriarum
+              <lb ed="#C"/>capitulo XV</ref> <quote xml:id="bHY6yh-d41894-Qd1e123">Cunctis diebus suis impius super<lb ed="#C" break="no"/>bit 
+              et numerus annorum incertus est tyrannidis 
+              <lb ed="#C"/>eius sonitus terroris eius semper in auri<lb ed="#C" break="no"/>bus 
+              eius illius et cum pax sit semper in<lb ed="#C" break="no"/>sidiis 
+              suspicatur</quote>
+            </p>
+            <p xml:id="bHY6yh-d41894-d1e137">
+              ¶ <ref xml:id="bHY6yh-d41894-Rd1e133">Idem 34 capitulo</ref> <quote xml:id="bHY6yh-d41894-Qd1e139">Dominus reg<lb ed="#C" break="no"/>nare 
+              facit ypocritam propter peccata populi</quote>
+            </p>
+            <p xml:id="bHY6yh-d41894-d1e146">
+              ¶ <ref xml:id="bHY6yh-d41894-Rd1e146">Cronica de nugis philosophurum capitulo VIII de fia<lb ed="#C" break="no"/>ndro </ref>
+              <quote xml:id="bHY6yh-d41894-Qd1e150">Periander corintius vnus de septem 
+              <lb ed="#C"/>sapientibus ait debentes securos tiramp<pb ed="#C" n="57-v"/><cb ed="#C" n="a"/><lb ed="#C"/>nos 
```

Comment:
Here you should still add break=no for the line break. Otherwise it looks great.

```suggestion
              <lb ed="#C"/>sapientibus ait debentes securos tiramp<pb ed="#C" n="57-v"/><cb ed="#C" n="a"/><lb ed="#C" break="no"/>nos 
```